### PR TITLE
Track ktlint's code styles

### PR DESCRIPTION
**Pull request by swankjesse**
Continuation indent is now 2 instead of 4.

### CODE REVIEW

**Comment by oldergod** [comment]
already in conflict 🙀

**Review by oldergod — APPROVED** [review]

File: configs/codestyles/Square.xml [hunk 1]
```diff
@@ -294,24 +294,20 @@
...
-    <option name="EXTENDS_LIST_WRAP" value="5" />
-    <option name="METHOD_CALL_CHAIN_WRAP" value="1" />
-    <option name="ASSIGNMENT_WRAP" value="1" />
     <option name="METHOD_ANNOTATION_WRAP" value="1" />
     <option name="FIELD_ANNOTATION_WRAP" value="1" />
     <option name="ENUM_CONSTANTS_WRAP" value="2" />
     <indentOptions>
       <option name="INDENT_SIZE" value="2" />
-      <option name="CONTINUATION_INDENT_SIZE" value="4" />
+      <option name="CONTINUATION_INDENT_SIZE" value="2" />
```
Comment (vRallev) [inline configs/codestyles/Square.xml:309]: That would have a big impact on many other codebases. We'd basically need to reformat all of them.

**Comment by JakeWharton** [comment]
Change `SquareAndroid`, too, please. Or just delete it and resolve #45.

File: configs/codestyles/Square.xml (hunk 1 above)
Comment (vRallev) [reply configs/codestyles/Square.xml:309]: If your concern is only KtLint, then there's an open ticket to bring back the continuation indent: https://github.com/pinterest/ktlint/issues/816

**Comment by rjrjr** [comment]
Retracted my 👎 . Chatted offline, and realized that we can put a tag on the current commit to give ourselves control over rolling this out across our code base.

Bonus points if you go ahead and make that tag for us all.

**Comment by swankjesse** [comment]
I’m gonna hold on this until we can roll forward with something that’s less invasive.

**Pull request closed without merging**